### PR TITLE
Module 2 Webpack

### PR DESCRIPTION
**Pull request by UltramarineDev**

### CODE REVIEW

File: netflix-roulette/src/app/app.component.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (omelenetskiy) [inline netflix-roulette/src/app/app.component.js]: folder -> App
components -> App.jsx

Comment (omelenetskiy) [reply netflix-roulette/src/app/app.component.js]: src/components/App/App.jsx

File: netflix-roulette/src/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="de">
+
+<head>
+  <base href="/">
+  <title>Netflix Roulette</title>
+  <meta charset="utf-8">
+</head>
+
+<body>
+  <div class="content"></div>
```
Comment (omelenetskiy) [inline netflix-roulette/src/index.html]: id="content" || "root"

File: netflix-roulette/src/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import App from './app/app.component';
```
Comment (omelenetskiy) [inline netflix-roulette/src/index.js]: `import {render} from 'react-dom';`

Comment (omelenetskiy) [reply netflix-roulette/src/index.js]:
```
render(
  <App/>,
  document.getElementById('root'),
);
```

**Review by omelenetskiy — APPROVED** [review]

File: netflix-roulette/src/app/app.component.js (hunk 1 above)
Comment (UltramarineDev) [reply netflix-roulette/src/app/app.component.js]: done.

File: netflix-roulette/src/index.html (hunk 2 above)
Comment (UltramarineDev) [reply netflix-roulette/src/index.html]: done.

File: netflix-roulette/src/index.js (hunk 3 above)
Comment (UltramarineDev) [reply netflix-roulette/src/index.js]: done.

**Pull request merged**